### PR TITLE
 ZHECKmEx: update profile page

### PR DESCRIPTION
**Pull request by AmritSidhu**
Replaced existing table and fields to match the prototype
Removed old data that was potentially used for testing and that is unnecessary
Localised all the text on the profile page
Added back button on profile page

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -184,9 +186,9 @@ en:
       menu: Menu
       warning: Warning
       message: "%{message}"
-      profile: "Profile"
-      teams: "Team Management"
-      events: "Events Viewer"
+      profile: Your Profile
```
Comment (jakubmiarka) [inline config/locales/en.yml]:
```suggestion
      profile: Your profile
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -121,23 +121,25 @@ en:
...
-    assume_roles_heading: "Switch Roles"
-    assume_roles_legend: "Select which role(s) you wish to assume?"
+    title: Your profile
+    item: Item
+    details: Your details
+    action: Action
+    name: Name
+    email: Email Address
+    password: Password
+    last_changed: Last changed X days ago
```
Comment (jakubmiarka) [inline config/locales/en.yml]: we don't have this functionality and will not be available in the near future, can we just drop it?

File: app/views/profile/show.html.erb [hunk 3]
```diff
@@ -1,95 +1,69 @@
...
+      <th class="govuk-table__header govuk-visually-hidden" scope="col"><%= t 'profile.action' %></th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell"><%= t 'profile.name' %></td>
+      <td class="govuk-table__cell"><%= @user.full_name %></td>
+      <td class="govuk-table__cell"></td>
+    </tr>
+    <tr class="govuk-table__row">
```
Comment (jakubmiarka) [inline app/views/profile/show.html.erb]: the table is not fully responsive when the email is long. can we make it break lines properly?
e.g.
image

File: app/views/profile/show.html.erb [hunk 4]
```diff
@@ -1,95 +1,69 @@
...
+      <td class="govuk-table__cell"><%= t 'profile.email' %></td>
+      <td class="govuk-table__cell"><%= @user.email %></td>
+      <td class="govuk-table__cell"></td>
+    </tr>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell"><%= t 'profile.password' %></td>
+      <td class="govuk-table__cell"><%= t 'profile.last_changed' %></td>
+      <td class="govuk-table__cell"><%= link_to t('profile.change'), profile_change_password_path %></td>
```
Comment (jakubmiarka) [inline app/views/profile/show.html.erb]:
```suggestion
      <td class="govuk-table__cell"><%= link_to t('profile.change'), profile_change_password_path, class: 'govuk-link' %></td>
```

File: app/views/profile/show.html.erb (hunk 3 above)
Comment (jakubmiarka) [reply app/views/profile/show.html.erb]: the email is meant to break on next line so the Change button is not off-screen

Comment (AmritSidhu) [reply app/views/profile/show.html.erb]: Strange i dont have that issue locally, the change button displays on screen
Screen Shot 2019-10-17 at 10 06 01

Comment (jakubmiarka) [reply app/views/profile/show.html.erb]: I guess your email is shorter, try changing it in the console to mine one and see.

Comment (AmritSidhu) [reply app/views/profile/show.html.erb]: Yeah i see now, have pushed up a fix

File: spec/system/visit_profile_page_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+RSpec.describe 'Profile page', type: :system do
+  before(:each) do
+    login_user
+  end
+
+  context 'profile page' do
+    it 'should show profile page' do
+      visit profile_path
+      expect(page).to have_content 'Your profile'
```
Comment (jakubmiarka) [inline spec/system/visit_profile_page_spec.rb]:
```suggestion
      expect(page).to have_content t('profile.title')
```

**Pull request merged**